src/monorepo-shipit/config/__tests__/testExportedPaths.js

```diff
@@ -36,10 +31,6 @@ export default function testExportedPaths(
     if (output === undefined) {
       expect(...outputDataset.getDiffs()).toBeUndefined();
     } else {
-      // 1. verify the path we are testing actually exists (and therefore the test makes sense)
-      expect(fs.existsSync(path.join(monorepoRoot, input))).toBe(true);
```

Comment:
This was stupid because it breaks easily without us being able to catch it on CI (unrelated workspace changes its structure which doesn't trigger Shipit tests). It would have to be a scanner that is meant to overcome this CI limitation.